packages/utils/src/utils/date-time.test.ts

```diff
@@ -0,0 +1,95 @@
+import { DateTime } from './date-time';
+
+const { now, nowPlusOneSecond } = {
```

Comment:
NIT obs — this is a funny one; why not just use plain declarations? 😄

You'd have saved one line plus a few chars, btw.

```ts
const now = {
  tai64: '4611686020108779339',
  unixSeconds: 1681391398,
  unixMilliseconds: 1681391398000,
  isoString: '2023-04-13T13:09:58.000Z',
}

const nowPlusOneSecond = {
  tai64: '4611686020108779340',
  unixSeconds: 1681391399,
  unixMilliseconds: 1681391399000,
  isoString: '2023-04-13T13:09:59.000Z',
}
```

Reply:
You didn't need to, but thanks!